### ISSUE TITLE
ability to name your hotbars

### ISSUE THREAD

**Issue by qbjl** [VideoGameSmash12/HotbarsPlus#32]
this is another good organization tweak, which can help sort your saved inventories if you like doing that.

**Comment by VideoGameSmash12** [comment]
I actually _really_ like this idea, but my main concern is that this might break vanilla compatibility. 

With that aside, I take it that in practice would be as simple as replacing the "Saved Hotbars" text in the creative menu with a custom one using information stored in the actual hotbar file.

![image](https://github.com/VideoGameSmash12/HotbarsPlus/assets/8041766/587c2c74-1e23-43d3-8da4-f15188dbe4c1)

**Comment by qbjl** [comment]
yes, but replace the text with an input so that it's much simpler. the text inside the input changes when you go from page to page, and updates when you type in it etc etc. that would make it easier for people to rename hotbars.

**Comment by qbjl** [comment]
and vanilla SHOULD be fine because the code probably will ignore an extra tag at the bottom and just use what's provided.

**Comment by VideoGameSmash12** [comment]
This could be expanded upon further beyond naming hotbars as well. It could be evolved into an entire metadata system, with descriptions, authors, etc.

**Comment by VideoGameSmash12** [comment]
This was implemented in Librarian, the successor to Hotbars+. Since that isn't out yet, this issue will remain open until Librarian gets officially released.

**Comment by VideoGameSmash12** [comment]
Done - https://github.com/VideoGameSmash12/Librarian
